client/js/test/table-view.test.js

```diff
@@ -11,6 +11,20 @@ describe('table view', () => {
 		document.documentElement.innerHTML = html;
 	});
 
+	describe('add row button', () => {
+		it('adds another row to the table body', () => {
+			const model = new TableModel(3, 3);
+			const view = new TableView(model);
+			view.init();
+
+			expect(model.numRows).toEqual(3);
```

Comment:
There is more to test here. Where should the Add Row button add a new row? Top of table? Bottom of table?